tests/page-objects/contacts/contacts.wdio.page.js

```diff
@@ -259,6 +260,11 @@ const getContactMedicID = async () => {
   return (await contactMedicID()).getText();
 };
 
+const createNewPerson = async () => {
+  await newPersonContactButton().waitForDisplayed();
+  await newPersonContactButton().click();
```

Comment:
It is the same thing, I don't know why I didn't see that was already implemented.
I removed that and use the method `addPerson` instead and removed the file `new-person.wdio.page.js` because it was not needed anymore.
Thank you for seeing this! 😄